src/main/java/com/code_intelligence/jazzer/Jazzer.java

```diff
@@ -180,7 +198,15 @@ private static String prepareArgv0(Map<String, String> additionalEnvironment) th
           "Setting environment variables in the wrapper is only supported on POSIX systems");
     }
     char shellQuote = isPosix() ? '\'' : '"';
-    String launcherTemplate = isPosix() ? "#!/usr/bin/env sh\n%s $@\n" : "@echo off\r\n%s %%*\r\n";
+    String launcherTemplate;
+    if (isAndroid()) {
+      launcherTemplate = "%s $@\n";
```

Comment:
Where does the shell live under Android? Could we make changes to the posix template so that it also works on Android?

Reply:
/system/bin/sh 

I'll update this to: /system/bin/env sh